projects/plugins/social/src/js/components/support-section/index.js

```diff
@@ -0,0 +1,33 @@
+import { Text } from '@automattic/jetpack-components';
+import { ExternalLink } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { Icon, lifesaver } from '@wordpress/icons';
+import { STORE_ID } from '../../store';
+import IconTextSection from '../icon-text-section';
+import styles from './styles.module.scss';
+
+const SupportSection = () => {
+	const isPremiumPlan = useSelect( select => select( STORE_ID ).isPremiumPlan() );
+
+	return isPremiumPlan ? (
+		<IconTextSection
+			icon={ <Icon icon={ lifesaver } size={ 30 }></Icon> }
```

Comment:
I think it might be nice to use an early return here to avoid the ternary, and make it easier to add more conditions down the road.

```js
if ( ! isPremiumPlan ) {
	return null;
}

return <IconTextSection ...
```